DemoApp/StreamRuntimeCheck+StreamInternal.swift

```diff
@@ -15,4 +15,10 @@ extension StreamRuntimeCheck {
         guard let intValue = Int(value) else { return nil }
         return LogLevel(rawValue: intValue)
     }
+    
+    static var subsystems: LogSubsystem? {
+        guard let value = ProcessInfo.processInfo.environment["STREAM_LOG_SUBSYSTEM"] else { return nil }
+        guard let intValue = Int(value) else { return nil }
+        return LogSubsystem(rawValue: intValue)
+    }
```

Comment:
For making it easy to only get API requests and responses to the console.
E.g. this gives me only curl styled logs for every request we make with `APIClient`
- STREAM_LOG_LEVEL = 0
- STREAM_LOG_SUBSYSTEM = 4